construct-info/ABC-sequences.fasta

```diff
@@ -0,0 +1,6 @@
+>ABC1
+^GCGTA
+>ABC2
+^ATAGC
+>ABC3
+^GTGCA
```

Comment:
Writing `>` in the sequence is not something included in fasta format, I would suggest adding this in the get_abcs function.

Reply:
Now I don't quite follow, do you mean that we should add '>' in the get_abc function and keep it out of the original file?